src/content/doc-tutorials/integrate-auth0-as-authentication-provider.mdx

```diff
@@ -72,18 +72,17 @@ To add custom claims, you must [create an Auth0 action](https://auth0.com/docs/c
 For this example, you can use the following code:
 
 <Tabs groupId="version">
-  <TabItem value="V1" label="Using Scope and Token" default>
+  <TabItem value="V2" label="Using DEFINE ACCESS" default>
```

Comment:
I see in the preview this sort of looks like any user can choose between one or the other when it's a version-specific thing. How about something like `Using DEFINE ACCESS (v. 2.x)` and `Using Scope and Token (v. 1.x)`?

Reply:
Since this information is at the start of the tutorials I think it covers the version information <img width="1117" alt="Screenshot 2024-10-09 at 06 24 44" src="https://github.com/user-attachments/assets/c3e50693-c547-4484-8d1c-ae424b98a4c1">

Reply:
Okay, sounds good!